pyproject.toml

```diff
@@ -173,16 +170,23 @@ pybamm = [
     "pybamm/plotting/mplstyle",
 ]
 
-[tool.setuptools.packages.find]
-include = ["pybamm", "pybamm.*"]
+[tool.scikit-build.cmake]
+version = ">=3.13"
+verbose = true
+build-type = "Release"
+
+[[tool.scikit-build.overrides]]
+if.any.env.BUILD_IDAKLU = true
+if.any.env.CIBUILDWHEEL = true
```

Comment:
I think we should decide on a format for this – whether to set it to `true`, `True`, or `ON`. I think `ON/OFF` sounds nice, but it is a matter of preference.

Reply:
Yes, `ON/OFF` sounds nice